src/Controller/Admin/AdminController.php

```diff
@@ -59,10 +56,8 @@ public function includeConferenceAction(): Response
         ]);
     }
 
-    /**
-     * @Route(path="/participation/accept", name="participation_accept")
-     * @Security("has_role('ROLE_ADMIN')")
-     */
+    #[Route(path: '/participation/accept', name: 'participation_accept')]
+    #[Security('has_role("ROLE_ADMIN"')]
```

Comment:
:heart_eyes:

Reply:
you must not check for a specific role here, but let the security component resolve everything for you.
So basically: 
```suggestion
    #[Security('ROLE_ADMIN')]
```

this comment applies everywhere else